project/Build.scala

```diff
@@ -399,6 +399,19 @@ object Build {
       )
     }
 
+    def withScalaJSTestNGPlugin: Project = {
+      project.settings(
+        scalacOptions in Test ++= {
+          if (isGeneratingEclipse) {
+            Seq.empty
+          } else {
+            val jar = (packageBin in (testNGPlugin, Compile)).value
+            Seq(s"-Xplugin:$jar")
+          }
+        }
+      )
```

Comment:
We should probably have method `private def withInternalScalaTestPlugin(project: Project)` to factor out code from `withScalaJSJUnitPlugin` and `withScalaJSTestNGPlugin`.